extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -820,9 +811,10 @@ export class AmpA4A extends AMP.BaseElement {
             }))
         .then(status => {
           checkStillCurrent();
-          if (getMode().localDev &&
-              this.element.getAttribute('type') == 'fake') {
-            // do not verify signature for fake type ad
+          if (this.element.getAttribute('type') == 'fake' &&
```

Comment:
We might as well move this before the signatureVerifierFor promise given we don't actually care about the result.

Reply:
Moved.